jepsen-tools/collect_abandoned_metrics.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+FILE=$DIR/pod_ips
+IPS=$(cat "$FILE")
+PORT=8080
+METRIC_NAME="insolar_requests_abandoned{"
+
+for ip in $IPS
+do
+    metric=$(curl -s "$ip":$PORT/metrics | grep $METRIC_NAME)
+    if [ -n "$metric" ]
+    then
+        echo "$ip":"$metric"
+    fi
+done
```

Comment:
Please avoid using bash scripts for... anything. There is Python3 available on all pods.

Reply:
there will be .py after changing the docker image

Reply:
Nope, it's already there:
```
ssh -o 'StrictHostKeyChecking no' -i ./base-image/id_rsa -p 32002 gopher@localhost
$ python3 --version
Python 3.5.2
```